### ISSUE TITLE
Document multipath operation?

### ISSUE THREAD

**Issue by huitema** [marten-seemann/draft-seemann-quic-address-discovery#6]
In a multipath environment, the client is likely to use different addresses on different paths, either because the paths are bound to different addresses or network interfaces at the client, or because different paths are mapped differently by various NAT. It is interesting to discover on which path an address was observed.

I think this is possible with the current description:
~~~
OBSERVED_ADDRESS Frame {
       Type (i) = 0x9f81a2..0x9f81a3,
       Sequence Number (i),
       [ IPv4 (32) ],
       [ IPv6 (128) ],
       Port (16),
   }
~~~
The client has to remember on which path a particular `REQUEST_ADDRESS` was sent, and can then match that to the sending path.

So, not really an issue, except maybe it could be discussed in an "operation consideration" or "multipath consideration" section.

**Comment by marten-seemann** [comment]
It's not only needed in QUIC Multipath, vanilla QUIC also requires the requester to remember the path. The frames are intentionally defined as probing frames, so you can send them in probing packets during the path validation phase, and you might be probing multiple paths in parallel.

**Comment by huitema** [comment]
I think that we need a bit of editing there. The current text says (section 4.1 Observed Address frame):
~~~
Request ID:
    The request identifier of the request for which this response is intended.
~~~
.. but there is no request defined, thus no ID to copy.

If the multipath extension is negotiated, it might be nice to stick there a path-id that identifies the path using the observed address, and maybe stipulate to set the path-id to 0 if the multipath extension is not negotiated.

It might also be nice to have some kind of counter, separate from the path ID, to distinguish new values from old packets repeated because of loss recovery.

**Comment by huitema** [comment]
I think the issue is addressed by PR #14 (Remove request ID) and PR  #15 (Clarify that retransmissions happen on same path).
